prime-router/metadata/schemas/covid-19.schema

```diff
@@ -780,6 +804,28 @@ elements:
     documentation: |
       The date which the specimen was collected. The default format is yyyyMMddHHmmsszz
 
+  - name: specimen_collection_method
```

Comment:
For this and a few other similar fields being added here and used by ND, I'm wondering how will we generate these from SimpleReport data?

SimpleReport is currently sending us these vaguely similar fields:
```
specimen_source_site_code
specimen_type
equipment_model_name
test_performed_code
```
Is it possible to use those instead?  If not, then we need to have a way to map them.

Reply:
Those don't really appear to map to specimen_collection_method, but I'll look and see if we can map somehow.

Reply:
They all seem to be various descriptions of nasal passages to me.  Very appealing to the middle-schooler in me.